packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -460,13 +461,20 @@ export const UIModeView: React.FC<{}> = ({
           testModel={testModel}
           runTests={() => runTests('bounce-if-busy', visibleTestIds)} />
         <Toolbar noMinHeight={true}>
-          {!isRunningTest && !progress && <div className='section-title'>Tests</div>}
-          {!isRunningTest && progress && <div data-testid='status-line' className='status-line'>
-            <div>{progress.passed}/{progress.total} passed ({(progress.passed / progress.total) * 100 | 0}%)</div>
-          </div>}
-          {isRunningTest && progress && <div data-testid='status-line' className='status-line'>
-            <div>Running {progress.passed}/{runningState.testIds.size} passed ({(progress.passed / runningState.testIds.size) * 100 | 0}%)</div>
-          </div>}
+          {!progress ? (<StatusLine
```

Comment:
Put this as a single `<StatusLine>` reference with nullish coalescence and ternaries where that isn't acceptable?